src/module/helpers/settings.ts

```diff
@@ -123,29 +125,21 @@ export class IronswornSettings {
   static get logCharacterChanges(): boolean {
     return !!game.settings.get('foundry-ironsworn', 'log-changes')
   }
```

Comment:
It may be worth adding a static getter for the global-supply flag, to make the other call more concise:

```suggestion
  }
  
  static get globalSupply(): boolean {
    return game.settings.get('foundry-ironsworn', 'shared-supply') as boolean
  }
```